src/components/AddBook.js

```diff
@@ -1,12 +1,40 @@
-const AddBook = () => (
-  <div className="form-container">
-    <form>
-      <input className="input-text" type="text" required />
-      <input className="input-text" type="text" required />
-
-      <input className="input-submit" type="submit" value="ADD BOOK" />
-    </form>
-  </div>
-);
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const handleAddBook = (e) => {
+    e.preventDefault();
+
+    const title = e.target.elements.title.value;
+    const author = e.target.elements.author.value;
+
+    dispatch(addBook({ title, author }));
+  };
+
+  return (
+    <div className="form-container">
+      <form onSubmit={handleAddBook}>
+        <input
+          className="input-text"
+          type="text"
+          name="title"
+          placeholder="Title"
+          required
+        />
+        <input
+          className="input-text"
+          type="text"
+          name="author"
+          placeholder="Author"
+          required
+        />
+        <input className="input-submit" type="submit" value="ADD BOOK" />
```

Comment:
- [ ] I suggest that you use a `<Button></Button>` instead of `input` and also use event handler `onClick` to add a book to the books array 
```jsx
<Button onClick={() => dispatch(addBook({ title, author }))}>
        Add book
      </Button>
```